torch/csrc/jit/script/tree_views.h

```diff
@@ -23,10 +23,10 @@ namespace script {
 // Def   = Def(Ident name, List<Param> params, List<Stmt> body)         TK_DEF
 //
 // Stmt  = If(Expr cond, List<Stmt> true_body, List<Stmt> false_body)   TK_IF
-//       | For(List<Ident> targets, List<Expr> iters, List<Stmt> body)  TK_FOR
+//       | For(List<Expr> targets, List<Expr> iters, List<Stmt> body)   TK_FOR
 //       | While(Expr cond, List<Stmt> body)                            TK_WHILE
 //       | Global(List<Ident> idents)                                   TK_GLOBAL
-//       | Assign(List<Ident> lhs, AssignType maybe_reduce, Expr rhs)   TK_ASSIGN
+//       | Assign(List<Expr> lhs, AssignType maybe_reduce, Expr rhs)    TK_ASSIGN
```

Comment:
Do we now have an invariant that `lhs` can be a list of `Var`s only? Please add a comment listing the expected kinds of expressions that can appear in that subtree